server/data/apps.json

```
@@ -1930,5 +1960,5 @@
         "elementURL": "https://bitbadger.solutions/open-source/myweblog/rss.html#podcast-feeds"
       }
     ]
-  }
+  }  
```

I would remove the extra space too
```suggestion
  } 
```